### ISSUE TITLE
Custom JWT secret path not recognized from 1.30.0

### ISSUE THREAD

**Issue by privatejava** [NethermindEth/nethermind#7912]
**Description**
I am running a Ethereum mainnet node and it is giving me issue from v.1.30.0  which does not respect the `JsonRpc.JwtSecretFile` path custom provided in config.

**Steps to Reproduce**
In order to replicate the behavior, please provide a detailed list of steps:
1. Install v1.30.0 nethermind
2. Use custom path for  `JsonRpc.JwtSecretFile`  like `/conf/jwt.hex`
3. Then after runnnig the node it shows error like this 
```
nethermind  | 14 Dec 21:15:48 | Starting Ethash block producer & sealer 
nethermind  | 14 Dec 21:15:48 | Generating authentication secret... 
nethermind  | 14 Dec 21:15:48 | The authentication secret hasn't been found in '/nethermind/keystore/jwt-secret'so it has been automatically created. 
nethermind  | 14 Dec 21:15:48 | 
```

**Actual behavior**
I believe it has some bug with configuration because v.1.29.1 was working all good.

**Expected behavior**
It shoudl simply show like this 
```
nethermind  | 14 Dec 21:12:21 | Reading authentication secret from '/conf/jwt.hex' 
nethermind  | 14 Dec 21:12:21 | 
```

**Screenshots**
![image](https://github.com/user-attachments/assets/c38de416-b6ee-4389-a97f-951e195daf60)

**Desktop**
 - Operating System: Ubuntu 22.04
 - Installation Method: Docker
 - Consensus Client: v.1.30.0

**Comment by Scooletz** [comment]
Could you share the command line parameters, especially the `JsonRpc.JwtSecretFile` so that we can verify it?

**Comment by ssonthal** [comment]
@privatejava @Scooletz I believe this issue was created to add this log when default jwt secret file is not required -> #7746 

https://github.com/NethermindEth/nethermind/issues/7746

**Comment by privatejava** [comment]
> Could you share the command line parameters, especially the `JsonRpc.JwtSecretFile` so that we can verify it?

I am using docker with config file

_docker-compose.yml_
```yaml
nethermind:
    ulimits: 
      nofile:
        soft: 1000000
        hard: 1000000
    container_name: nethermind
    restart: unless-stopped
    image: nethermind/nethermind:${NETHERMIND_VERSION}
    hostname: nethermind_${NET}
    ports:
      - 24001:8545/tcp
      - 24002:8551/udp
    volumes:
      - ./data/nethermind:/nethermind/data
      - ./config/nethermind/main.cfg:/nethermind/configs/mainnet.cfg
      - ./config/nethermind:/conf
    command: |-
            --datadir data
```

_main.cfg_
```json
{
    "Init": {
        "ChainSpecPath": "chainspec/foundation.json",
        "GenesisHash": "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        "BaseDbPath": "nethermind_db/mainnet",
        "LogFileName": "mainnet.logs.txt",
        "MemoryHint": 2048000000,
	"StateDbKeyScheme": "HalfPath"
    },
    "Network": {
        "ActivePeersMaxCount": 100
    },
    "Sync": {
	"FastSync": true,
        "SnapSync": true,
        "PivotNumber": 19050000,
        "PivotHash": "0x01be0f86a10ea554774cf6a5d8c8e4ad33fbe87eb1e5b8951bced2611f079db9",
        "PivotTotalDifficulty": "58750003716598352816469",
        "SnapServingEnabled": true,
        
        
        "FastSyncCatchUpHeightDelta": "10000000000"
    },
    "EthStats": {
        "Server": "wss://ethstats.net/api"
    },
    "Metrics": {
        "NodeName": "Mainnet"
    },
    "Blocks": {
        "TargetBlockGasLimit": 30000000
    },
    "JsonRpc": {
        "Enabled": true,
        "Timeout": 20000,
        "Host": "0.0.0.0",
        "EngineHost": "0.0.0.0",
	"MaxBatchSize": 10000,
        "EnginePort": 8551,
        "Port": 8545,
        "JwtSecretFile": "/conf/jwt.hex",
	"EnabledModules": ["eth","subscribe","trace","txpool","web3","personal","proof", "net", "parity","health","rpc", "admin"],
        "AdditionalRpcUrls": [
            "http://localhost:8551|http;ws|admin|net;eth;subscribe;engine;web3;client"
        ]
    },
    "Merge": {
        "Enabled": true
    },
    "HealthChecks":{
        "Enabled": true,
        "UIEnabled": true,
	"LowStorageSpaceWarningThreshold": 16
    },
    "Pruning":{
	"AvailableSpaceCheckEnabled": true,
        "FullPruningMemoryBudgetMb" : 8000,
	"FullPruningTrigger" : "VolumeFreeSpace",
	"Mode": "Hybrid",
	"FullPruningThresholdMb" : 350000,
	"FullPruningCompletionBehavior": "None"
    }
}
```

_jwt.hex_
```
9bcf5b683b9934f548568bb1cc3ecc607f46d8cef514f637f87056ea36690dc1
```

**Comment by Scooletz** [comment]
Clearly what @ssonthal stated above. As #7765 has been merged after the release of `13.0.1` this will be shipped with the next minor / bug fix.
